.rdoc_options

```diff
@@ -11,3 +11,13 @@ rdoc_include:
 exclude:
 - lib/irb
 - .gemspec
+
+autolink_excluded_words:
+- Class
+- Method
+- Module
+- Process
+- RDoc
+- Set
+
+canonical_root: https://docs.ruby-lang.org/en/3.4
```

Comment:
I think this should be set to https://docs.ruby-lang.org/en/master` as we want master to always be the version that shows up first in search results?

Reply:
I don’t understand. This only applies to Ruby 3.4’s doc, not master’s. Should all versions set it to master doc’s url?